docs/operations/faq/hc_grpc.rst

```diff
@@ -0,0 +1,79 @@
+Health Checking your gRPC service
+=================================
+
+Currently, Envoy does not have a properly defined gRPC health check type. Therefore, what several deployments do is
+create an alternate HTTP endpoint on a different port and use that as the health check endpoint. Then, the sidecar envoy's
+configuration can expose routing to the gRPC endpoint and the HTTP endpoint pivoting on content-type. For example here is
+an excerpt of the route table:
```

Comment:
Instead of duplicating this, can we just link into the example config in the repo where I think we have this? (I don't have a strong preference here but seems better to not duplicate and potentially just document pertinent snippets).